### PR TITLE
Adds railcontent_id to group by fetch all.

### PR DESCRIPTION
**Pull request by DidierF**

### CODE REVIEW

File: src/services/sanity.js [hunk 1]
```diff
@@ -518,6 +518,7 @@ export async function fetchAll(brand, type, {
         const lessonsFilter = `_type == '${type}' && brand == '${brand}' && ^._id == ${groupBy}._ref ${searchFilter} ${includedFieldsFilter} ${progressFilter}`;
         entityFieldsString = `
                 'id': _id,
+                railcontent_id,
```
Comment (roxanariza) [inline src/services/sanity.js:521]: In this case (grouped by artist), the railcontent_id will always be null.

File: src/services/sanity.js [hunk 2]
```diff
@@ -534,6 +535,7 @@ export async function fetchAll(brand, type, {
         const lessonsFilter = `brand == '${brand}' && ^._id in ${groupBy}[]._ref ${typeFilter} ${searchFilter} ${includedFieldsFilter} ${progressFilter}`;
         entityFieldsString = `
                 'id': _id,
+                railcontent_id,
```
Comment (roxanariza) [inline src/services/sanity.js:538]: In this case (grouped by genre), the railcontent_id will always be null.

**Review by roxanariza — APPROVED** [review]
Your updates look good!
For group_by(artist/genre) the railcontent_id will be null always, but I supposed that you want the response structure to be the same; so I am fine with this.

I am not sure why we need "instructor" and "coaches" on fetchLessonContent. Are duplicate data, but with different fields

File: src/services/sanity.js (hunk 2 above)
Comment (DidierF) [reply src/services/sanity.js:538]: In some cases (workouts, for example), group by instructor is using this case, so railcontent_id is the instructor's railcontent_id. 🤔

Comment (roxanariza) [reply src/services/sanity.js:538]: Yes, when we have group by instructor the railcontent_id will be set.

**Pull request merged**